### ISSUE TITLE
[Question] Did you use MOD in pretrain stage?

### ISSUE THREAD

**Issue by zjr2000** [MCG-NJU/p-MoD#3]

**Comment by jonasjunzhang** [comment]
Hi @zjr2000 , thank you for your interest in our work! 

We only use MoD in SFT stage. We tried inserting MoD during pretraining (while freezing the LLM) but the loss diverges.

We think it might be worth trying to add MoD in pretraining and unfreeze the LLM. However, this approach increases training costs and makes it difficult to ensure a fair comparison with the LLaVA baseline. As a result, we leave this exploration for future work.

**Comment by zjr2000** [comment]
Thank you for your response and for clarifying!
